.changeset/thirty-forks-share.md

```diff
@@ -0,0 +1,5 @@
+---
+"@tiptap/extension-youtube": major
```

Comment:
This should be `patch` instead of `major`

Reply:
Got it! I've changed to `patch`